core/src/impl/Kokkos_Profiling.hpp

```diff
@@ -125,15 +125,20 @@ void syncDualView(const std::string& label, const void* const ptr,
 void modifyDualView(const std::string& label, const void* const ptr,
                     bool on_device);
 
-void initialize();
+void initialize(const std::string& = {});
 void finalize();
+bool printHelp(const std::string&);
+void parseArgs(const std::string&);
+void parseArgs(int, char**);
```

Comment:
Why do we need to expose this overload?

Reply:
- `void parseArgs(const std::string&)` for internal use
- `void parseArgs(int, char**)` for the user to call directly
  - I added this in the original version of the testing bc it excessively tedious to convert to a string when calling directly

Reply:
I know I'm slow on this, but I just checked, and it doesn't look the tests call these. Do we still need both? Could we remove the int,char** guy?